src/go/k8s/pkg/resources/statefulset_test.go

```diff
@@ -66,67 +71,74 @@ func TestEnsure(t *testing.T) {
 		{"update replicas", stsResource, replicasUpdatedCluster, replicasUpdatedSts},
 		{"update resources", stsResource, resourcesUpdatedCluster, resourcesUpdatedSts},
 		{"disabled sidecar", nil, noSidecarCluster, noSidecarSts},
+		{"cluster without shadow index cache dir", stsResource, withShadowIndexCacheDirectory, stsWithSecondPersistentVolume},
```

Comment:
so is this with or without shadow index? 🤔 test name says without but variable used in the test says with

Reply:
My bad. I will change the name of the variables